Userland/Services/WebContent/PageHost.h

```diff
@@ -75,6 +77,7 @@ class PageHost final : public Web::PageClient {
 
     RefPtr<Core::Timer> m_invalidation_coalescing_timer;
     Gfx::IntRect m_invalidation_rect;
+    String m_preferred_color_scheme { "auto" };
```

Comment:
Does this need to be a string? Does the IPC mechanism not have enums?

Reply:
Enums do work! I've switched to using one.